### ISSUE TITLE
Bank Negara Indonesia Stock Analysis

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#501]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Bank Negara Indonesia Stock Analysis
:red_circle: **Aim** : The aim of this project is to analyze the stocks mentioned in the dataset using Machine Learning models.
:red_circle: **Dataset** : https://www.kaggle.com/datasets/caesarmario/bank-negara-indonesia-stock-historical-price
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by pawaspy** [comment]
@abhisheks008 can you assign me this for JWoC. I have already created project on this topic so it will be very fast for me.
Full name : Pawas Pandey
GitHub Profile Link : githyb.com/pawaspy
Participant ID (If not, then put NA) : NA
Approach for this Project : I will use lstm and different models such as random forest or linear regression or svm for prediction.
What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.) : JWoC

**Comment by abhisheks008** [comment]
JWOC will start from Jan 15, issues will be assigned thereafter.

**Comment by Yuvika-14** [comment]
hi abhisekh
Full name: Yuvika Singh
git hub profile link:https://github.com/Yuvika-14/Yuvikademo.git
approch: ensemble methods, gradient boosting, neural networks.
Take care of the missing data if there are any categorical values then will use one hot encoder or label encoder depending upon the issue.

**Comment by abhisheks008** [comment]
> @abhisheks008 can you assign me this for JWoC. I have already created project on this topic so it will be very fast for me. Full name : Pawas Pandey GitHub Profile Link : githyb.com/pawaspy Participant ID (If not, then put NA) : NA Approach for this Project : I will use lstm and different models such as random forest or linear regression or svm for prediction. What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.) : JWoC

Issue assigned to you @pawaspy 

Models to be implemented are,
1. Random Forest
2. Decision tree
3. Logistic
4. MLP
5. XgBoost
6. Gradient Boosting
7. Lasso
8. Ridge
